pkg/client/unversioned/request.go

```diff
@@ -841,10 +841,9 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) Resu
 
 	// Did the server give us a status response?
 	isStatusResponse := false
-	var status *unversioned.Status
-	result, err := runtime.Decode(r.content.Codec, body)
-	if out, ok := result.(*unversioned.Status); err == nil && ok && len(out.Status) > 0 {
-		status = out
+	status := &unversioned.Status{}
```

Comment:
Add a comment that this is for compatibility with servers that do not return proper version info on status so someone doesn't come along and refactor